docs/source/user_guide/connector/connector_authorization.rst

```diff
@@ -0,0 +1,185 @@
+
+==================================================
+Authorization schemes supported by Connector
+==================================================
+
+.. toctree::
+   :maxdepth: 2
+
+Overview
+========
+
+Connector supports the most used authorization methods in Web APIs:
+
+- API Key
+- OAuth 2.0 "Client Credentials" and "Authorization Code" grants.
+
+Let's review them in detail.
+
+------------------
+
+API Key
+========
+
+Depending on the Web API specification, you must send your API Key as a bearer token, a parameter in the URL (query parameter), or a parameter in the request header. Let's review these cases in detail:
+
+Bearer Token
+==============
```

Comment:
Is the 'Bearer Token' at the same level with 'API Key'?

Reply:
No, it is under the API Key section. Adjusted for more clarity.